src/components/safe-messages/Msg/styles.module.css

```diff
@@ -1,21 +1,6 @@
-.toggle {
-  font-weight: 700;
-  text-decoration: underline;
-  text-align: left;
-}
-
-.truncated {
-  display: -webkit-box;
-  -webkit-line-clamp: 3;
-  -webkit-box-orient: vertical;
-  overflow: hidden;
-  max-height: 9ex;
-}
-
-.overflow {
-  overflow: auto;
-}
-
-.pre {
-  margin: 0;
+.msg :global .MuiInputBase-input {
+  white-space: pre;
+  font-family: monospace;
+  font-size: 0.85rem;
```

Comment:
This ends up being 13.6px for me. I would suggest we use an explicit pixel value like 14px.

Reply:
Whilst I agree, this copies the sizing from the Safe Apps display. Would you suggest we change it in both places? I'd say `12/13px` instead of `14px` in which case.

Reply:
If its like this in Safe Apps I would leave it like this for now.